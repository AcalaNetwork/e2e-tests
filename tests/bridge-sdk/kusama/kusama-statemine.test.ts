import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'kusama',
  to: 'statemine',
  token: 'KSM',
})
