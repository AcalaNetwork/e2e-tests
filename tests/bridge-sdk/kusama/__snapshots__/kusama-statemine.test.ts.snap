// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'kusama' to 'statemine' using bridgeSDK cross-chain 'KSM' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": "(rounded 8)",
  "toChain": "(rounded 12)",
}
`;

exports[`'kusama' to 'statemine' using bridgeSDK cross-chain 'KSM' > fee 1`] = `"(rounded 0.000004)"`;

exports[`'kusama' to 'statemine' using bridgeSDK cross-chain 'KSM' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 10,
  "toChain": 10,
}
`;
